test/config/training_service.yml

```diff
@@ -95,6 +95,8 @@ pai:
     containerNFSMountPath: 
     paiStorageConfigName: 
 remote:
+  remoteConfig:
+    reuse: false
```

Comment:
so there is no test for "reuse" is true?

Reply:
no, the default value is false, and this value will set to true when pipeline set `$(remote_reuse)`.